src/qtt/measurements/scans.py

```diff
@@ -349,6 +349,10 @@ def _initialize_live_plotting(alldata, plotparam, liveplotwindow=None, subplots=
         pyqtgraph.mkQApp().processEvents()  # needed for the parameterviewer
     return liveplotwindow
 
+def _dataset_record_label(scanjob):
+    """ Return label to be used in dataset record """
```

Comment:
The argument `scanjob` en return value are not documented yet.

Reply:
> Some minor changes requested. In the future it would be nice to change the `scanjob` into a class. Then the `dataset_record_label`, etc. can become a property of `scanjob`.

Yes i agree. The reason it is not yet a a class is that serialization is an issue then.

Reply:
Method was private. I will add docs anyway